homeassistant/components/aquacell/sensor.py

```diff
@@ -77,6 +77,12 @@ class SoftenerSensorEntityDescription(SensorEntityDescription):
             "low",
         ],
     ),
+    SoftenerSensorEntityDescription(
+        key="last_update",
+        translation_key="last_update",
+        device_class=SensorDeviceClass.TIMESTAMP,
+        value_fn=lambda softener: softener.lastUpdate,  # type: ignore[arg-type, return-value]
```

Comment:
```suggestion
        value_fn=lambda softener: softener.lastUpdate,
```
You should instead amend the type hints on:
https://github.com/home-assistant/core/blob/477635d34fd787628e271ada0f0898d42ae02306/homeassistant/components/aquacell/sensor.py#L31
and
https://github.com/home-assistant/core/blob/477635d34fd787628e271ada0f0898d42ae02306/homeassistant/components/aquacell/sensor.py#L120

Reply:
Thanks!

Reply:
Thanks! Done.